src/partials/header.hbs

```diff
@@ -0,0 +1,45 @@
+<!-- header starts -->
+<header>
+  <nav>
```

Comment:
Why so many containers? NAV tag should have UL and LI so your logo can be inside LI element only

Reply:
it is because of the header layout in design